tests/testthat/test-generate.R

```diff
@@ -990,7 +990,7 @@ test_that("can generate system with aliased array", {
 
 test_that("can generate system with length and sum of aliased array", {
   dat <- odin_parse({
-    update(x) <- sb + lb
+    update(x) <- sb + lb + a[1]
```

Comment:
`a` was now correctly treated as unused; previously it was wrongly recorded as dependency in dim(b) <- dim(a) - but we never used it in a way that would have exposed the bug.